Camera_config:
  focal_lenghts:
    # x: 536.32
    # y: 536.34
    # x: 1070.27001953125
    # y: 1070.72998046875
    # x: 370.55255126953125
    # y: 376.434326171875
    x: 411.628
    y: 414.166

  optical_centers:
    # x: 633.545
    # y: 355.6885
    # x: 943.6300048828125
    # y: 529.5230102539062
    x: 447.032
    y: 206.403
    # y: 255.34486389160156

  distorsion_coefficients:
    k1: 0 
    k2: 0
    p1: 0
    p2: 0

  rotation_vector:
    x: 0.0
    y: 0.0
    z: 0.0

  translation_vector:
    x: 0.0
    y: 0.0
    z: 0.0

  camera_measures:
    camera_width: 896
    camera_height: 512